woocommerce-gateway-stripe.php

```diff
@@ -478,6 +478,16 @@ public function add_gateways( $methods ) {
 					}
 				}
 
+				// Don't mark Link as enabled if we're in the admin so it doesn't show up in the checkout editor page.
+				if ( is_admin() ) {
+					$methods = array_filter(
+						$methods,
+						function( $method ) {
+							return ! is_a( $method, 'WC_Stripe_UPE_Payment_Method_Link' );
```

Comment:
Nit-pick: I would use `WC_Stripe_UPE_Payment_Method_Link::class` instead, just for a more straightforward lookup and to avoid issues with class renaming.

Reply:
Good catch!